reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Windows.md

```diff
@@ -32,76 +32,122 @@ Be noted that when using the ZIP archive, you won't get the prerequisites check
 So in order for remoting over WSMan to work properly on Windows versions prior to Windows 10,
 you need to make sure the [prerequisites](#prerequisites) are met.
 
+## Deploying on Windows IoT
+
+Windows IoT already comes with Windows PowerShell which we will use to deploy PowerShell Core 6.
+
+- Create `PSSession` to target device
+
+  ```powershell
+  $s = New-PSSession -ComputerName <deviceIp> -Credential Administrator
+  ```
+
+- Copy the zip package to the device
```

Comment:
Should use ZIP consistently unless used as part of a file version in a code block.

Reply:
Fixed this everywhere we reference "ZIP package" or we are talking about the package. Left it lowercase in filenames and where "zip" and "unzip" refer to action/commands.